From 668a5a89109526e39540174b4a8a56574d0d09bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Thu, 15 Aug 2024 03:46:12 +0200
Subject: [PATCH] Drop versioned certifi dependency

Let it pick the version from the distribution.
---
 requirements/base.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/requirements/base.txt b/requirements/base.txt
index de9cbaab17..bce14bc510 100644
--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -7,8 +7,6 @@ PyYAML
 MarkupSafe
 requests<2.32.0 ; python_version < '3.10'
 requests>=2.32.3 ; python_version >= '3.10'
-certifi==2023.07.22; python_version < '3.10'
-certifi>=2024.7.4; python_version >= '3.10'
 distro>=1.0.1
 psutil>=5.0.0
 packaging>=21.3
-- 
2.45.2

